index.html

```diff
@@ -28,5 +29,163 @@ <h3 class="conn">LET’S CONNECT</h3>
             <li><a href="#"><img src="./images/Vector (3).png"></a></li> 
           </ul>
     </section>
+    <section class="grid-container">
+       <div class="grid-1">
+         <img src="./images/Snapshoot Portfolio.png" alt="project1" class="project1">
+         <h1 class="ton">Tonic</h1>
+            <div class="canopy">
+                <h3>canopy</h3>
+                <span>
+                    <img src="./images/Counter.png">
+                    back end dev
+                </span>
+                <span>
+                    <img src="./images/Counter.png">
+                    2015
+                </span>
+            </div>
+            <p class="bio">
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required.
+            </p>
+                <ul class="categories">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JAVASCRIPT</li>
+                </ul>
+            <button class="see-project">See Project</button>
+       </div> 
+       <div class="grid-2">
+        <img src="./images/Snapshoot Portfolio (1).png" alt="project2" class="project2">
+        <h1 class="ton">Tonic</h1>
+           <div class="canopy">
+               <h3>canopy</h3>
+               <span>
+                   <img src="./images/Counter.png">
+                   back end dev
+               </span>
+               <span>
+                   <img src="./images/Counter.png">
+                   2015
+               </span>
+           </div>
+           <p class="bio">
+               A daily selection of privately personalized reads;
+               no accounts or sign-ups required.
+           </p>
+               <ul class="categories">
+                   <li>HTML</li>
+                   <li>CSS</li>
+                   <li>JAVASCRIPT</li>
+               </ul>
+           <button class="see-project">See Project</button>
+      </div> 
+
+      <div class="grid-3">
+        <img src="./images/Snapshoot Portfolio (2).png" alt="project3" class="project3">
+        <h1 class="ton">Tonic</h1>
```

Comment:
- Kindly make sure you have only one h1 tag in your HTML for the main title
for others, title and subtitle use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. :+1: 
   * This way will be semantically correct and the screen readers and other technologies will thank you.